README.md

```diff
@@ -95,6 +100,47 @@ User.create(
 )
 ```
 
+## Troubleshooting
+
+If you are getting errors such as `FATAL: role "user" doesn't exist. Couldn't create database.` while running `rake db:setup` or `rake db:create` then it may mean that your database is password protected. There are two ways to setup required databases:
+  
+1. (Recommended) Loosen local Postgres database security to allow local users without password
+
+    You have to edit [pg_hba.conf](https://www.postgresql.org/docs/9.6/static/auth-pg-hba-conf.html) config file (`/etc/postgresql/9.6/main/pg_hba.conf` on Unix) and add or update authorization line for local logins from `md5` to `trust`:
+    
+    ```
+    # "local" is for Unix domain socket connections only
+    local   all             all                                     trust
+    # IPv4 local connections:
+    host    all             all             127.0.0.1/32            trust
+    ```   
+    
+    Create Postgres superuser that will link to your account:
+    ```
+    sudo -u postgres createuser `whoami` -ds
+    ```
+    
+    Now `bundle exec rake db:setup` command should work.
+    
+2. Create users and databases manually
+
+    ```sh
+    sudo -u postgres psql -c "CREATE USER wmdb_dev WITH PASSWORD 'wmdb_dev';"
+    sudo -u postgres createdb -O wmdb_dev wmdb_dev -E utf-8
```

Comment:
I totally missed this in the earlier review, but can you please use `web-monitoring-db_development` for the DB name? I’d like to make sure the DB someone winds up with by following this matches as closely as possible to the one they’d wind up with if they did the normal steps (`rake db:setup`).